internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -302,7 +304,8 @@ func (provisioner *KopsProvisioner) ProvisionCluster(cluster *model.Cluster, aws
 		return errors.Wrap(err, "failed to delete PodDisruptionBudget calico-typha")
 	}
 
-	// Need to remove two items from the metrics-server because the fields after the creation are imutable so the
+	//TODO: Remove this block after we roll-out to prometheus-adapter to production
```

Comment:
add a space after the // (for all comments you made)

ie:
```suggestion
	// TODO: Remove this block after we roll-out to prometheus-adapter to production
```